ansible/playbook.yml

```
@@ -0,0 +1,65 @@
+---
+- name: Install tools
+  hosts: nodes
+  become: true
+  tasks:
+    - name: Install Git
+      yum:
+        name: git
+        state: present
+        update_cache: yes
+    - name: Install Python 3
+      yum:
+        name: python3
+        state: present
+        update_cache: yes
```

Very minor, but worth noting that we can combine these if we want to:
```suggestion
    - name: Install Git & Python 3
      yum:
        name: 
          - git
          - python3
        state: present
        update_cache: yes
```